packages/exo/docs/exo-taxonomy.md

```diff
@@ -1,28 +1,32 @@
 # A Taxonomy of Exo-making Functions
 
-An Exo is a Far object protected by an interface guard. We chose the term "exo" both because
-* "exo" means "outside". An exo object is reachable from outside the vat. Other vats can hold a reference (a capability) to an exo, and anyone with a reference to an exo can send it messages. Put another way, the exo objects are the objects in a vat that are outward facing, and so must be fully defensive against receiving bad messages.
-* Alludes to "ExoSkeleton", a protective outside layer that is an effective first defense against many threats. Likewise, an exo's interface guard is a great first layer of type-like (pattern-based) input validation. The programmers remaining burden to make the outward facing exo objects fully defensive thus becomes easier.
+An Exo is a Far object protected by an interface guard. We chose the term "exo" because
+* It means "outside". An exo object is reachable from outside the vat. Other vats can hold a reference (a capability) to an exo, and anyone with a reference to an exo can send it messages.
+* It alludes to "ExoSkeleton", a protective outside layer that is an effective first defense against many threats. Likewise, an exo's interface guard is a great first layer of type-like ([pattern](https://github.com/endojs/endo/tree/HEAD/packages/patterns)-based) input validation protecting against many kinds of bad messages. The programmer's remaining burden to make the exo objects fully defensive thus becomes easier.
```

Comment:
"HEAD" is guaranteed to always refer to the primary branch, while "master" refers specifically to a branch (or other git ref, for that matter) named "master" whether or not it is primary. GitHub implements redirects after renaming "master" to e.g. "main", but those go away if a new "master" ref is ever created, which is highly plausible if someone fat-fingers a `git push`. So basically, "HEAD" is just more reliable.

Reply:
Cool. In that case, suggestion withdraw. Resolving. (LGTM still stands)

Reply:
Not worth another commit but I request that we stick to `master` for permalinks.

- it's consistent with the 68 of 68 other instances of GH URLs in this codebase
- when you [ask Github for a permalink](https://docs.github.com/en/repositories/working-with-files/using-files/getting-permanent-links-to-files), that's what it gives you
- if we ever change it from `master`, Github will redirect (as you point out)
  - if we change to `main` and someone accidentally pushes `master` we will delete the `master` branch as it would be very confusing to have both
- the code browser UI is clearer when on `master` than `HEAD` because it shows you what branch you're on:

`HEAD`
<img width="649" alt="Screenshot 2023-01-27 at 6 11 59 AM" src="https://user-images.githubusercontent.com/21505/215107250-79da9f40-9ee1-4be5-88b8-3b9a61f44d3d.png">

`master`
<img width="665" alt="Screenshot 2023-01-27 at 6 12 46 AM" src="https://user-images.githubusercontent.com/21505/215107408-eaabae58-d231-4497-b2f1-b9c0e09c2fc5.png">

Reply:
FYI, I just reported that issue at https://github.com/orgs/community/discussions/45563 .